back-end/API/tests/test_admin_jobOffer.py

```
@@ -0,0 +1,227 @@
+    "offerDebut": "2021-12-12",
+    "active": True,
+    "approbationMessage": "Super offre!",
+    "employerId": 1,
+    "isApproved": True,
+    "approvedDate": datetime.now(),
+    "last_modified_by_id": 1
+}
+
+def VerifyData(jobOfferData):
+    for data in job_offer1_data:
+        if (data not in jobOfferData) and (type(jobOfferData[data]) != type(job_offer1_data[data])):
```

Ruff (0.15.2)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)